### ISSUE TITLE
명사 추출기에서 기구축된 사전을 seed dictionary로 활용

### ISSUE THREAD

**Issue by hungry-wook** [lovit/soynlp#131]
통계/학습 기반 명사 추출기에 대해서도
기구축된 사전을 베이스로 하여 명사 후보를 늘려나갈 수 있도록 기능을 제공하면 좋을것 같습니다.

**Comment by lovit** [comment]
네 좋은 기능이라고 생각합니다. 특히 L-R graph 에서 모호한 명사들을 미리 "명사라고 지정"할 수 있다면 그 이후에 추출되는 다른 명사들의 정확도가 올라갈 것으로 기대됩니다.

**Comment by lovit** [comment]
PR이 병합되어 완료.
